### PR TITLE
Update my solutions

### PR DESCRIPTION
**Pull request by lpercival**
Starting with Python, not done yet. Need input on how to organize solutions (see trello)

### CODE REVIEW

File: javascript/solutions/part1/LongestWordFinder.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+var sentenceString = "This is a long sentence with words of multiple lengths for inputting into your function.";
+var longestWord = '';
+
+//note that this is imperfect for punctuation
```
Comment (helin24) [inline javascript/solutions/part1/LongestWordFinder.js:4]: Ha yes. I might list this with others as 'handle more edge cases' in free-form part of problem statement.

File: python/solutions/part1/longest_word_finder.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+sentence_string = "This is a long sentence with words of multiple lengths for inputting into your function"
+longest_word = ''
+
+words = sentence_string.split()
```
Comment (helin24) [inline python/solutions/part1/longest_word_finder.py:4]: for clarity, swap lines 2 and 4? I think of the splitting as something to be done right away with the sentence to get an array, and then the longest_word initialization should happen immediately before the loop to emphasize that it'll be changed in the loop.

**Review by helin24 — APPROVED** [review]
Looks great!

**Comment by helin24** [comment]
Go ahead and merge this and other PRs that are hanging out unless you're planning specific other changes. I'll make new PRs for the items I mentioned potentially wanting to add later.

File: python/solutions/part1/longest_word_finder.py (hunk 2 above)
Comment (lpercival) [reply python/solutions/part1/longest_word_finder.py:4]: I see no difference in clarity, just a stylistic preference. Mine is to declare variables first before doing any work, so this is how I would write it. You may change it if you want.

Comment (helin24) [reply python/solutions/part1/longest_word_finder.py:4]: I hadn't thought about it that way - works for me!

**Pull request merged**